socrata/socrata_upload.py

```diff
@@ -12,7 +12,7 @@
 
 # Connect to Athena
 cursor = connect(
-    s3_staging_dir=str(os.getenv("AWS_ATHENA_S3_STAGING_DIR")) + "/",
+    s3_staging_dir="s3://ccao-athena-results-us-east-1/",
```

Comment:
**[Question, non-blocking]** If we're not going to read `AWS_ATHENA_S3_STAGING_DIR` from the environment, do we need to pass it in as an env var in the workflow definition above?

Reply:
I was getting an error that the staging directory wasn't a proper S3 location. My only thought was that perhaps I'm not passing the script an environmental var that actually exists?

Reply:
@jeancochrane This still causes an error for some reason... to you know if the github var actually exists?